asciidoc/header.adoc

```diff
@@ -1,7 +0,0 @@
-++++
```

Comment:
it looked really shitty without this styling change, is that not a problem anymore?